docs/overview.rst

```diff
@@ -15,52 +15,68 @@ abilities. It is meant to be **easy to use** and **easy to self-host**.
 Why use Kinto?
 ==============
 
-We believe data belongs to the users, and not to the application authors. When
-writing applications, data should be made available to any device, connected or
-offline, and everything should be in sync.
-
-Rather than spending a non-trivial amount of time and expertise on implementing
-all that (and then maintaining it!), you could use Kinto, that does all that
-for you:
-
-- Expose your data over an HTTP interface, unlike databases like PostgreSQL
-- Use simple HTTP requests rather than SQL
-- Use `Kinto.js <https://kintojs.readthedocs.io/en/latest/>`_ to easily
-  implement offline first clients
-- Choose the database you want from those that Kinto supports, and use a
-  unified API to access its data
-- Manage your data using the handy
-  `admin UI <http://kinto.github.io/kinto-admin/>`_
-- Easily
-  :ref:`set up live push notifications <tutorial-notifications-websockets>`
-  for live updates of your application
-- Make it possible to
-  :ref:`share data between users <api-permissions>`
-  (using fine-grained permissions)
-- Take advantage of
-  :ref:`schema validation <collection-json-schema>`
-  if you need it
-- :ref:`Track changes <api-history>` on your application data
+    Don't build silos. Redecentralize the Web!
+
+It's hard for frontend developers to respect users privacy when building applications
+that work offline, store data remotely, and synchronise accross devices.
+Existing solutions either rely on big corporations that crave for user data, or require
+a non-trivial amount of time and expertise to setup a new server for every new project.
+
+We want to help developers focus on the frontend, and we don't want the challenge
+of storing user data to get in their way! The path between a new idea and starting into
+production should be short!
+
+Also, we believe data belongs to the users, and not to the application authors.
```

Comment:
Historically, "data" has been the plural of "datum", but I personally use it as an "uncountable" noun -- "some data" -- which typically takes singular verbs. "The data is clear." I think this might be different in British English, but I'm not sure. I think either way is OK.